common/src/main/java/dev/latvian/mods/kubejs/block/custom/BasicBlockJS.java

```diff
@@ -31,6 +32,7 @@
 import net.minecraft.world.entity.LivingEntity;
 import net.minecraft.world.entity.player.Player;
 import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.Items;
```

Comment:
Looks like some random imports got added, you can run 'Optimise Imports' in Intellij to clean them up.